tower_cli/models/base.py

```diff
@@ -115,10 +115,11 @@ def __new__(cls, name, bases, attrs):
 
         # Ensure that the endpoint ends in a trailing slash, since we
         # expect this when we build URLs based on it.
-        if not newattrs['endpoint'].startswith('/'):
-            newattrs['endpoint'] = '/' + newattrs['endpoint']
-        if not newattrs['endpoint'].endswith('/'):
-            newattrs['endpoint'] += '/'
+        if isinstance(newattrs['endpoint'], basestring):
```

Comment:
That's true. It will fail python3, where `basestring` is removed. This is also the reason for travis test failures. Try using `six.text_type` or whatever is suitable.